pkg/reconciler/serverlessservice/resources/services.go

```diff
@@ -81,6 +84,50 @@ func MakePublicEndpoints(sks *v1alpha1.ServerlessService, src *corev1.Endpoints)
 	}
 }
 
+// MakeServiceEntry constructs a Istio ServiceEntry that is to create inbound listeners in advance.
+// ServiceEntry itself is not used as an access point.
+func MakeServiceEntry(sks *v1alpha1.ServerlessService, src *corev1.Endpoints) *v1alpha3.ServiceEntry {
+	return &v1alpha3.ServiceEntry{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      sks.Name,
+			Namespace: sks.Namespace,
+			Labels: resources.UnionMaps(sks.GetLabels(), map[string]string{
+				networking.SKSLabelKey:    sks.Name,
+				networking.ServiceTypeKey: string(networking.ServiceTypePublic),
+			}),
+			Annotations:     resources.CopyMap(sks.GetAnnotations()),
+			OwnerReferences: []metav1.OwnerReference{*kmeta.NewControllerRef(sks)},
+		},
+		Spec: istiov1alpha3.ServiceEntry{
+			Hosts:    []string{"placeholder" + "." + sks.Name + "." + sks.Namespace + ".svc"}, // This host name must be unique in the cluster.
+			Location: istiov1alpha3.ServiceEntry_MESH_INTERNAL,
+			Ports: []*istiov1alpha3.Port{{
+				Number:   80, // This port is not used.
+				Protocol: "http",
+				Name:     "http1",
+			}},
+			Resolution: istiov1alpha3.ServiceEntry_STATIC,
+			Endpoints:  FilterEndpoints(src),
+		},
```

Comment:
My understanding was that we put both source ans destination ips in the service entry?